### ISSUE TITLE
Recent API changes appears to have broken downloading

### ISSUE THREAD

**Issue by ScherisMarie** [frozenpandaman/mangadex-dl#50]
```
scheris@scheriss-mbp Downloads % ./mangadex-dl.py
mangadex-dl v0.6
Enter manga URL or ID: https://mangadex.org/title/024cfbe3-9bc8-49bf-8ca5-f9f49ae98f3d/sweet-suffering

TITLE: Sweet Suffering
Available chapters:
 Oneshot

Enter chapter(s) to download: oneshot

Downloading chapter Oneshot...
Traceback (most recent call last):
  File "/Users/scheris/Downloads/./mangadex-dl.py", line 285, in <module>
    dl(manga_id, lang_code, args.cbz, args.datasaver, args.outdir)
  File "/Users/scheris/Downloads/./mangadex-dl.py", line 187, in dl
    chaphash = chapter["data"]["attributes"]["hash"]
KeyError: 'hash'
```

Terminal output included above, but should be the same with any other manga on the site. Data saver flag being there or not doesn't appear to make a difference.

Seems like the hash and data/data-saver information that used to be grabbed in `chapter["data"]["attributes"]` doesn't exist in this call anymore?

EDIT: Looks like specifically **v5.4.1** of the API is what caused this, of which there was a one-week grace period before they removed it from what the current code uses.

**Comment by Ingvix** [comment]
Not sure when @frozenpandaman gets around to fix it/review my pull request but if anyone's impatient a fix is already included in my fork's [reducerequests-dev](https://github.com/Ingvix/mangadex-dl/tree/reducerequests-dev) branch along with some output beautification changes. Functionality is the same.

**Comment by frozenpandaman** [comment]
fixed in #48 (thank you!)
